composer.json

```diff
@@ -9,6 +10,16 @@
     "require": {
         "jalno/lumen": "^0.0.1"
     },
+    "require-dev": {
+        "nunomaduro/larastan": "^0.7.6",
```

Comment:
Why you needed larastan?

Reply:
As our last conversation about `nunomaduro/larastan`, I think I should use it for all packages.\
I removed it and use `phpstan/phpstan` instead.